src/main/java/com/start/learn/employeeinheritancehuman/Employee.java

```diff
@@ -0,0 +1,119 @@
+        this.salary = salary;
+        hireDate = LocalDate.of(yearOfHireDay, monthOfHireDay, dayOfHireDay);
+        salaryDate = LocalDate.now().withDayOfMonth(15);
+        this.occupation = occupation;
+        employeeExperience = 0;
+        promotionPercent = 0;
+    }
+
+    public LocalDate getHireDate() {
+        return hireDate;
+    }
+
+    public LocalDate getSalaryDate() {
+        return salaryDate;
+    }
+
+    public double getSalary() {
+        return salary;
+    }
+
+    public double getEmployeeExperience() {
+        return employeeExperience;
+    }
+
+    public double getPromotionPercent() {
+
+        return promotionPercent;
+    }
+
+    public void salaryReview() {
+        employeeExperience = getCalculatedEmployeeExperience();
+        if (employeeExperience > 1) {
+            promotionPercent = 0;
+        }
+        if (employeeExperience >= 1 && employeeExperience < 2) {
+            promotionPercent = 25;
+        }
+        if (employeeExperience >= 2 && employeeExperience < 3) {
+            promotionPercent = 35;
+        }
+        if (employeeExperience >= 3 && employeeExperience < 4) {
+            promotionPercent = 45;
+        }
+        if (employeeExperience >= 4) {
+            promotionPercent = 80;
+        }
+        double raise = salary * promotionPercent / 100;
+        salary += raise;
+    }
+
+    @Override
+    public String toString() {
+        return "\nEmployee Name = " + getName() + ", Employee Surname = " + getSurname() +
+                "\nOccupation = " + occupation + ",Salary = " + salary + "\nHire Day := " + hireDate + ", " +
+                "\nEmployee_experience = " + getEmployeeExperience() + " year(s)" + ",Promotion Percentage: " + promotionPercent + "%" +
+                "\nUpcoming Salary Date: " + salaryDate;
+    }
+    //Calculate amount of Days until salary day
+    public long getDaysUntilSalaryInCurrentMonth() {
+
+        long daysUntilSalary = 0;
+        LocalDate todayDate = LocalDate.now();
+        LocalDate salaryDateInNextMonth = salaryDate.plusMonths(1);
+        if (todayDate.getDayOfMonth() == salaryDate.getDayOfMonth()) {
+            System.out.println("TODAY is salary day");
+        } else if (salaryDate.getDayOfMonth() > todayDate.getDayOfMonth()) {
+            daysUntilSalary = ChronoUnit.DAYS.between(todayDate, salaryDate);
+        } else {
+            daysUntilSalary = ChronoUnit.DAYS.between(todayDate, salaryDateInNextMonth);
+            salaryDate = salaryDateInNextMonth;
+        }
+        return daysUntilSalary;
+    }
+
+    public int getCalculatedEmployeeExperience() {
+        Period employeeExperience;
+        int calculatedEmployeeExperience = 0;
+        employeeExperience = hireDate.until(LocalDate.now());
+        calculatedEmployeeExperience = employeeExperience.getYears();
+        return calculatedEmployeeExperience;
+    }
+
+    @Override
+    public void getStatus() {
```

Comment:
return state because in name is "get" ;
String

Reply:
done